### DISCUSSION TITLE
19th October, 2021

### DISCUSSION THREAD

**Discussion by AngelOnFira** [rust-gamedev/wg#116] [Meetings]
Small meeting, figuring out how to get things started. I was the only one that showed up, but I hardly posted anywhere. I used the time to try and write down some process details.

**Action items:**

- Look through previous meeting minutes, learn from how things were done before
- Find a good way to prepare meetings
    - Github discussions
- Run a doodle poll to find ideal times for people

**Minutes:** https://hackmd.io/JdgmDD_ORHmPRRXX11GlYA

**Comment by kvark** [comment]
Somehow, I didn't realize there was a meeting! Thank you for showing up :heart:

**Reply by AngelOnFira** [reply]
It was probably because I only posted about it to a few channels, and didn't announce it anywhere :sweat_smile: part of my minutes from today was thinking about how I could improve outreach. More should know about it next time!
